core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationMain.scala

```diff
@@ -86,15 +86,23 @@ object QualificationMain extends Logging {
       eventLogFsFiltered
     }
 
-    if (filteredLogs.isEmpty) {
-      logWarning("No event logs to process after checking paths, exiting!")
-      return (0, Seq[QualificationSummaryInfo]())
-    }
+    // Update the status of event logs that were filtered above
+    val eventStatusFilteredMap = if(filteredLogs.size != eventStatusMap.size) {
+      EventLogPathProcessor.filterEventStatusMap(eventStatusMap, filteredLogs)
```

Comment:
This feels more like a hack. Creating a map of status, then reprocess-it again to remove filtered keys.
What is interesting here that we still leave files with invalid file extensions which may be part of a directory structure and users may not care about them.

Reply:
Refactor the code, so that it only adds event logs that failed in the previous step, in this step adds filtered status and in the next step (while processing application) adds event logs that were successful

Reply:
There will not be invalid files in the `statusReports`